test/TritonGPU/reduce-op-lowering.mlir

```diff
@@ -0,0 +1,41 @@
+// RUN: triton-opt %s --allocate-shared-memory --convert-triton-gpu-to-llvm | FileCheck %s
+
+// COM: Tests reduction when threads_per_warp(=16) < num_warps(=64). A possible case for Intel XPUs
```

Comment:
The comments are not aligned with the test case.
The test case is `threadsPerWarp=32`